implementation/src/main/java/io/smallrye/faulttolerance/CompositeObservableCommand.java

```diff
@@ -70,25 +69,14 @@ protected CompositeObservableCommand(Callable<? extends CompletionStage> callabl
         this.operation = operation;
         this.retryContext = retryContext;
         this.registry = registry;
-        this.queuedAt = System.nanoTime();
     }
 
     @Override
-    protected Observable construct() {
+    protected Observable<?> construct() {
         String metricsPrefix = MetricNames.metricsPrefix(operation.getMethod());
-
-        try {
-            if (registry != null && operation.hasBulkhead()) {
-                // TODO: in fact, we do not record the time spent in the queue but the time between command creation and command execution
-                histogramOf(metricsPrefix + MetricNames.BULKHEAD_WAITING_DURATION).update(System.nanoTime() - queuedAt);
-            }
-        } catch (Exception any) {
-            LOGGER.warn("Failed to update metrics", any);
-        }
-
+        Observable<Object> observable = Observable.create(
         // the retry metrics collection here mirrors the logic in HystrixCommandInterceptor.executeCommand
         // and MetricsCollectorFactory.MetricsCollectorImpl.beforeExecute/afterSuccess/onError
```

Comment:
If the comment here is now _under_ `Observable.create`, should probably increase indentation.